src/i18n/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+find_program(GETTEXT_MSGFMT_EXECUTABLE msgfmt)
+if(NOT GETTEXT_MSGFMT_EXECUTABLE)
+    message(FATAL_ERROR "msgfmt not found")
+endif()
+
+find_program(GETTEXT_MSGMERGE_EXECUTABLE msgmerge)
+if(NOT GETTEXT_MSGMERGE_EXECUTABLE)
+    message(FATAL_ERROR "msgmerge not found")
+endif()
+
+find_program(GETTEXT_MSGATTRIB_EXECUTABLE msgattrib)
+if(NOT GETTEXT_MSGATTRIB_EXECUTABLE)
+    message(FATAL_ERROR "msgattrib not found")
+endif()
+
+SET(TRANSLATIONS
+    de
+    en
+    es
+    it
+    ru
+)
+
+foreach(_trans ${TRANSLATIONS})
+add_custom_command(
+   COMMENT "Generate Locale '${_trans}'"
+   OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/Locale/${_trans}.po"
+   POST_BUILD
+   COMMAND mkdir -p Locale/${_trans}/LC_MESSAGES
+   COMMAND ${GETTEXT_MSGATTRIB_EXECUTABLE} --translated "${CMAKE_CURRENT_SOURCE_DIR}/Locale/${_trans}.po" -o "${CMAKE_CURRENT_SOURCE_DIR}/Locale/${_trans}.po"
+   COMMAND ${GETTEXT_MSGMERGE_EXECUTABLE} --update --no-fuzzy-matching "${CMAKE_CURRENT_SOURCE_DIR}/Locale/${_trans}.po" "${CMAKE_CURRENT_SOURCE_DIR}/Locale/en.po"
```

Comment:
Could this be the intention?
msgcat --use-first %{_trans}.po en.po -o %{_trans}.po
So that any untranslated strings use the English if available?